plasma-roundup/draft-plasma-article.md

```diff
@@ -0,0 +1,76 @@
+# Plasma Roundup: From MVP to Plasma Group
+By Chris Spannos
+
+Word count: 914
+
+Ethereum second layer scaling technology has come a long way in a very short period of time. Second layer solutions, innovations beyond the layer one protocol level, include State Channels, Side Chains and Plasma. Taken together, layer two technologies present a wide scope of possibility for scaling the Ethereum blockchain. This article focuses on Plasma development only.  
+
+In August 2017, Plasma creators Joseph Poon and Vitalik Buterin [proposed this framework](https://plasma.io/plasma.pdf) for scaling Ethereum transaction throughput to a "significant amount of state updates per second," potentially more than Paypal, Visa or other widely used merchant service providers.
+
+Aside from offering comparable, if not more, transactions per second Plasma stakes its security in the value of Ethereum’s decentralized mainchain rather than in a centralized merchant service.
+
+The promise of Plasma lays in its potential to help scale blockchain technology by processing a substantial amount of decentralized financial applications worldwide.
+
+In January 2018, Vitalik posted the [“Minimal Viable Plasma” (MVP) specification to the Ethereum Research Forums](https://ethresear.ch/t/minimal-viable-plasma/426). The specification was designed to offer simplicity and basic security properties to kickstart development. Teams immediately began building their own implementations.
+
+This overview is not meant to be exhaustive. Instead, by highlighting a few implementations, it aims to be indicative of the progress that Plasma has made over the past year.
+
+Although none of the Plasma models reviewed here are on mainnet or production ready, they demonstrate that this technology is not just theory. Taken together, the many implementations suggest that Plasma is moving rapidly toward realizing the scaling potential that its creators and implementers envision.     
+
+Among the early Plasma implementations was [FourthState Labs](https://github.com/FourthState/plasma-mvp-rootchain), whose design included a rootchain contract according to the Plasma MVP. This rootchain, which has been incorporated into other implementations, is pretty simple to test. To test, I used this environment:
+
+* Linux (Debian Stretch)
+* Truffle (v5.0.2)
+* Ganache CLI (v6.2.5)
+* NPM 6.7.0
+
+Follow these steps to test:
```

Comment:
@cspannos A lot of my comments might be more related to the state of these projects, than your work…

But why would I want to test? And looking into their code, I'm not even sure the test does anything constructive. How can I actually run things? Is that possible? I know their docs are lacking, but maybe we can fill that gap if you have any ideas at all?

Reply:
@ChrisChinchilla Great questions! I will take another look at their repo to see if anything can be added there. For now, my latest draft attempted to answer only at a very high level. If you think it is insufficient, however, I am absolutely fine if my reference to testing is deleted.